packages/node_modules/@webex/plugin-meetings/src/config.js

```diff
@@ -89,7 +89,8 @@ export default {
     enableExtmap: false,
     experimental: {
       enableMediaNegotiatedEvent: false,
-      enableUnifiedMeetings: false
+      enableUnifiedMeetings: true,
+      enableAdhocMeetings: true
```

Comment:
I guess we will need a function equivalent to `_toggleUnifiedMeetings()` for this new setting as the web app will need to set it based on feature toggles. Can you add it? It should be simpler - it only needs to change the config value and nothing else